### DISCUSSION TITLE
Simplify account id collection

### DISCUSSION THREAD

**Discussion by aaronfisher-code** [shirokumacode/overwatch-omnic-rewards#33] [Ideas]
Potentially might be simpler for new users to avoid the account ID process and use a battletag instead, as theres already a public API from the overwatch website which would allow for such a search feature to be implemented. You request a text input of the battletag, escape the '#' with '%23' and you can pull the id for the account without needing any authenticated logins as shown below for my battletag "Dab#11921"

![image](https://user-images.githubusercontent.com/79825913/185846010-15f7f5c6-a860-44af-b426-9758b4fc45b0.png)

API endpoint: https://playoverwatch.com/en-us/search/account-by-name/

Just a thought! Hope it helps

**Comment by shirokumacode** [comment]
This is really useful, I'll implement this as soon as I have time. I didn't see anyone using this when I created the app or maybe I just didn't explore playoverwatch.com deep enough.

Thanks for the tip, much appreciated
